File: app/src/main/java/android/matrix/ooredoo/qa/problems/java/TwoStrings.java
package android.matrix.ooredoo.qa.problems.java;

public class TwoStrings {

    static String twoStrings(String s1, String s2) {

        for(int i =0; i < s2.length(); i++) {
            if(s1.indexOf(s2.charAt(i)) >= 0) {
                return "YES";
            }
        }

        return "NO";

    }
}

File: app/src/main/java/android/matrix/ooredoo/qa/problems/java/TwoStrings.kt
package android.matrix.ooredoo.qa.problems.java

class TwoStrings {

    fun twoStrings(s1: String, s2: String): String {

        fun twoStrings(s1: String, s2: String): String {
            for (i in 0 until s2.length) {
                if (s1.indexOf(s2[i]) >= 0) {
                    return "YES"
                }
            }

            return "NO"

        }

    }
}